packages/fxa-settings/src/components/BentoMenu/index.tsx

```diff
@@ -27,18 +27,24 @@ export const BentoMenu = () => {
   const dropDownId = 'drop-down-bento-menu';
   const iconClassNames = 'inline-block w-5 -mb-1 ltr:pr-1 rtl:pl-1';
   const { l10n } = useLocalization();
+  const bentoMenuTitle = l10n.getString(
+    'bento-menu-title',
+    null,
+    'Firefox Bento Menu'
+  );
 
   return (
     <div className="relative self-center flex" ref={bentoMenuInsideRef}>
       <button
         onClick={toggleRevealed}
         data-testid="drop-down-bento-menu-toggle"
-        title={l10n.getString('bento-menu-title', null, 'Firefox Bento Menu')}
+        title={bentoMenuTitle}
+        aria-label={bentoMenuTitle}
```

Comment:
Q: What's the benefit to having both a `title` _and_ `aria-label`?

Reply:
Let me know if this has improved since last year, but afaik `aria-label` still has stronger screen-reader support than `title` (though I hope it gains full support soon). Since the bento menu button lacks any anchor text, visually impaired users who use a screen reader that does not support the `title` attribute will have no context of the button's purpose.

On the flip side, `aria-label` is completely non-visual, while most browsers will show a tooltip on hover when the `title` attribute is present, making it beneficial for visual users.

I recommend using `aria-label` in conjunction with `title` for wider support.

Reply:
> _"We probably want to do this all around then."_

Filed as #13282 so we don't lose track of this when/if we do a big a11y sprint.